src/cogent3/core/annotation_db.py

```diff
@@ -448,6 +480,39 @@ def _compatible_schema(db: sqlite3.Connection, schema: dict[str, set]) -> bool:
     return True
 
 
+def _rename_column_if_exists(
+    db: sqlite3.Connection, table_name: str, old_column: str, new_column: str
+) -> None:
+    """Rename a column in a sqlite3 database only if it exists.
+
+    Parameters
+    ----------
+    db : sqlite3.Connection
+        the sqlite3 connection
+    table_name : str
+        the table name
+    old_column : str
+        the column to rename if it exists
+    new_column : str
+        the new name of old_column
+    """
+    cur = db.cursor()
+    table_info = cur.execute(f"PRAGMA table_info({table_name});").fetchall()
+
+    # Check if the column exists
+    for column_info in table_info:
+        if column_info["name"] == old_column:
+            break
+    else:
+        return  # There is no column to rename
+
+    sql = f"""ALTER TABLE {table_name}
+RENAME COLUMN "{old_column}" TO {new_column}"""
```

Comment:
why split this across lines? If this is about needing to stay within line limits, you can do the following instead
```python
    sql = (f"ALTER TABLE {table_name} "
           f'RENAME COLUMN "{old_column}" TO {new_column}')
```

Reply:
Ah, just an old unnecessary habit. Fixed!